step-grid-parent/step-grid-api/src/main/java/step/grid/TokenWrapper.java

```diff
@@ -77,7 +77,7 @@ public TokenWrapperOwner getCurrentOwner() {
 		return currentOwner;
 	}
 
-	protected void setCurrentOwner(TokenWrapperOwner currentOwner) {
+	public void setCurrentOwner(TokenWrapperOwner currentOwner) {

```

Comment:
@david-stephan I assume these changes are not required anymore by SED-2056?

Reply:
@jeromecomte I'm not sure why it was switch to public at first. I don't see any need for it (no usage outside of package step.grid).
![image](https://github.com/exense/step-grid/assets/19704350/88d9663d-ff1f-4d5e-9355-bfc8d07123ba)

Reply:
@david-stephan I made setState public in order to invalidate the token. Since we're now exposing the invalidateToken of the grid I assume we don't need this anymore. setCurrentOwner and performAtomatically were made public to be consistent. All these 3 changes could be reverted now.